integration-tests/pom.xml

```diff
@@ -113,4 +113,44 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <!--
+         Integration test profile.
+         Explicitly triggered with -P integrationtest or by environment variable "integrationtest" set
+         -->
+        <profile>
+            <id>integrationtest</id>
+            <activation>
+                <property>
+                    <name>integrationtest</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <includes>
+                                <include>**/*IntegrationTest.java</include>
+                                <include>**/*Flow.java</include>
```

Comment:
I think we should change this to FlowTest.java and rename the test classes accordingly.

Reply:
Addressed